.github/workflows/docs.yml

```diff
@@ -3,7 +3,7 @@ name: Build and Deploy Documentation
 on:
   # Triggers the workflow on push or pull request events but only for the trunk branch
   push:
-    branches: [ trunk ]
+    branches: [ trunk, develop ]
```

Comment:
@cadic We may still want to generate docs only on push to the `trunk` brach, not for push to develop branch, right?

Reply:
@iamdharmesh thanks for noticing that, of course we should only generate docs for the trunk only (I believe this could be even changed to "publish a release" event). What do you think?

Reply:
Yes @cadic, "publish a release" is more appropriate here.